src/utilstrencodings.cpp

```diff
@@ -528,3 +528,39 @@ std::optional<std::vector<uint8_t>> ParseArbitraryInt(const std::string& num_str
     }
     return result;
 }
+
+std::string ArbitraryIntStr(std::vector<uint8_t> bytes)
```

Comment:
I'd like for there to be some tests for this function.

Reply:
Do we actually need to print diversifiers as integers, or can we just print them as little-endian hex byte arrays (i.e. 22 hex digits)?

Reply:
I thought I had some already, but turns out I didn't, and in fact adding them caught an ordering bug in the impl 😄

Reply:
@daira we need to print them as integers because we accept them in the RPC as integers.